app/javascript/map.js

```diff
@@ -0,0 +1,103 @@
+const center = { lat: 35.67337556092948, lng: 139.81827882218855 }; // 江東区の中心の緯度経度
+const select = document.getElementById('js-radius-select');
+const facilities = gon.searchInfo['facilities'];
+let markers = [];
+let infoWindows = [];
+
+function initMap() {
+  const map = new google.maps.Map(document.getElementById('map'), {
+    center: center,
+    zoom: 13,
+    mapTypeControl: false,
+    streetViewControl: false,
+    fullscreenControl: false
+  });
+
+  new google.maps.Marker({
+    map: map,
+    position: center,
+  });
+
+  for (let i = 0; i < facilities.length; i++) {
+    const coordinatesMarker = new google.maps.LatLng({ lat: parseFloat(facilities[i]['latitude']), lng: parseFloat(facilities[i]['longitude']) });
+    markers[i] = new google.maps.Marker({
+      position: coordinatesMarker,
+      map: map,
+      icon: 'https://maps.google.com/mapfiles/ms/icons/yellow-dot.png'
+    });
+
+    infoWindows[i] = new google.maps.InfoWindow({
+      content: `${facilities[i].name}<br><a href="/facilities/${facilities[i].id}">詳細ページ</a>`
+    });
+
+    markerEvent(i);
+  }
+
+  function markerEvent(i) {
+    markers[i].addListener('click', () => {
+      infoWindows[i].open(map, markers[i]);
+      for (let n = 0; n < facilities.length; n++) {
+        if (n === i) {
+          continue;
+        }
+        infoWindows[n].close();
+      }
+    });
+  }
+
+  circle = new google.maps.Circle({
+    map: map,
+    center: center,
```

Comment:
プロパティ名と変数名が同じ場合はこういう感じでスッキリ書けるので、こちらの書き方も覚えておくといいと
思います〜！
```suggestion
    map,
    center,
```

Reply:
都度直してしまったので特定のコミットを出せなくて申し訳ないのですが修正しました！